package.json

```diff
@@ -1,28 +1,28 @@
 {
     "name": "@elgato/icons",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

Comment:
As we use `npm version` on `main` to publish, this version should stay static as part of PRs.

Reply:
Just updated it :+1: